src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const strWithStyles = sourceString.split(';');
+  const objectCss = {};
```

Comment:
The variable name `objectCss` is not very descriptive. Consider renaming it to something more indicative of its purpose, like `styleObject`.